spec/features/front_end_spec.rb

```diff
@@ -54,6 +54,20 @@
       )
       expect(tailwind_config_file).to include('module.exports')
     end
+
+    context "with graphql" do
+      before(:all) do
```

Comment:
Beware of using before(:all) as it may cause state to leak between tests. If you are using rspec-rails, and use_transactional_fixtures is enabled, then records created in before(:all) are not automatically rolled back.